PWGLF/TableProducer/LFTreeCreatorNuclei.cxx

```diff
@@ -0,0 +1,177 @@
+                  full::NSigTPCPi,
+                  full::NSigTPCKa,
+                  full::NSigTPCPr,
+                  full::NSigTPCDe,
+                  full::NSigTPC3He,
+                  full::NSigTOFPi,
+                  full::NSigTOFKa,
+                  full::NSigTOFPr,
+                  full::NSigTOFDe,
+                  full::NSigTOF3He,
+                  full::TOFmatch,
+                  full::Px,
+                  full::Py,
+                  full::Pz,
+                  full::Pt,
+                  full::P,
+                  full::Eta,
+                  full::Phi,
+                  full::Sign);
+
+DECLARE_SOA_TABLE(LfCandNucleusFullEvents, "AOD", "LFNUCLEvent",
+                  collision::BCId,
+                  collision::NumContrib,
+                  collision::PosX,
+                  collision::PosY,
+                  collision::PosZ,
+                  full::IsEventReject,
+                  full::RunNumber);
+} // namespace o2::aod
+
+/// Writes the full information in an output TTree
+struct CandidateTreeWriter {
+  Produces<o2::aod::LfCandNucleusFull> rowCandidateFull;
+  Produces<o2::aod::LfCandNucleusFullEvents> rowCandidateFullEvents;
+
+  void init(o2::framework::InitContext&)
+  {
+  }
+
+  Configurable<float> yMin{"yMin", -0.5, "Maximum rapidity"};
+  Configurable<float> yMax{"yMax", 0.5, "Minimum rapidity"};
+  Configurable<float> cfgCutVertex{"cfgCutVertex", 10.0f, "Accepted z-vertex range"};
+  Configurable<float> cfgCutEta{"cfgCutEta", 0.8f, "Eta range for tracks"};
+  Configurable<float> nsigmacutLow{"nsigmacutLow", -8.0, "Value of the Nsigma cut"};
+  Configurable<float> nsigmacutHigh{"nsigmacutHigh", +8.0, "Value of the Nsigma cut"};
+
+  Filter collisionFilter = nabs(aod::collision::posZ) < cfgCutVertex;
+  Filter trackFilter = (nabs(aod::track::eta) < cfgCutEta) && (aod::track::isGlobalTrack == (uint8_t) true);
+  using TrackCandidates = soa::Filtered<soa::Join<aod::Tracks, aod::TracksExtra, aod::TracksExtended, aod::TrackSelection,
+                                                  aod::pidTPCFullPi, aod::pidTOFFullPi,
+                                                  aod::pidTPCFullKa, aod::pidTOFFullKa,
+                                                  aod::pidTPCFullPr, aod::pidTOFFullPr,
+                                                  aod::pidTPCFullDe, aod::pidTOFFullDe,
+                                                  aod::pidTPCFullHe, aod::pidTOFFullHe>>;
+
+  void process(soa::Filtered<soa::Join<aod::Collisions,
+                                       aod::EvSels>>::iterator const& collision,
+               TrackCandidates const& tracks)
+  {
+    // Filling event properties
+    rowCandidateFullEvents.reserve(collision.size());
+    rowCandidateFullEvents(
+      collision.bcId(),
+      collision.numContrib(),
+      collision.posX(),
+      collision.posY(),
+      collision.posZ(),
+      0,  // iseventselected //to be set
+      1); // runnumber //to be sets
```

Comment:
this you can probably set it via: `collision.bc().runNumber()`

Reply:
Hi @njacazio, I have set runnumber as suggested. 
On the folder to store the tree creator, I have chosen TableProducer as it is done similarly in HF.
We can move it to tasks, if you believe it is a better place.

Reply:
Hi @njacazio and @fbellini,
I would say that this task can stay in TableProducer since it indeed produces skims. However, for these skims to be used properly in a consumer task, one has to move the data model definition to a separate header, which can be visible to more than just this task. I guess this can be the topic of a future PR. 
I also see that this PR is modifying the aodMerger.cxx. Is this really intended ? Then maybe @jgrosseo should have a look.

Reply:
Thanks for flagging. This must be the result of an invalid merge. @fbellini please check before making the PR if you touch unexpected files. Github nicely shows the changes and one can scroll through...

Reply:
I think it sneaked in later when you pushed 
![image](https://user-images.githubusercontent.com/25029837/161515212-f67e5772-6076-47dc-9d12-64d86679c583.png)

Reply:
@jgrosseo, @iarsene hi! Thank you for spotting this mistake and sorry for messing up with my previous push. It was due to a misconfiguration of my fork. Hopefully my settings are ok now and it should not happen again. 
To answer @iarsene, yes, I can provide the separate header in a future PR. Thanks for the suggestion.